bin/lint-versions

```diff
@@ -11,5 +11,5 @@
 #
 # lint-versions - Check rust version
 
-grep "rust-version = " Cargo.toml | grep -q "1\.71\.0" || \
+grep "rust-version = " Cargo.toml | grep -q "1\.74\.0" || \
 (echo "Please validate new Rust versions for compilation time performance regressions or ask Team Testing to do so. Afterwards change the tested version in bin/lint-versions" && exit 1)
```

Comment:
I'm checking if the build time is ok

Reply:
thanks!

Reply:
On my system compile time went from 11:29 to 11:20, so all good!

Reply:
nice!